AUTHORS.rst

```diff
@@ -1,10 +1,19 @@
+
 ===============
 Main Developers
 ===============
 
-The following lists the authors (active ones are bold)
+The following lists the authors (active ones are bold) that contributed to upstream (or to some fork) of Perun:
+
+  * **Tomas Fiedor**: |github| `tfiedor <https://github.com/tfiedor>`_ |email| `TomasFiedor@gmail.com <mailto: TomasFiedor@gmail.com>`_ (founder), core
+  * **Jiri Pavela**: |github| `JiriPavela <https://github.com/JiriPavela>`_ |email| `JirkaPavela@gmail.com <mailto: JirkaPavela@gmail.com>`_ tracer, core
+  * **Ondrej Michal**: |github| `HarryMichal <https://github.com/HarryMichal>`_ energy profiler
```

Comment:
Maybe we can include even a link to the repository with the project?

https://gitlab.com/martymichal/sysrapl
```suggestion
  * **Ondrej Michal**: |github| `HarryMichal <https://github.com/HarryMichal>`_ [energy profiler](https://gitlab.com/martymichal/sysrapl)
```